addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2614,6 +2614,41 @@ export class OdooEditor extends EventTarget {
      *
      */
     initElementForEdition(element = this.editable) {
+        // Detect if the editable base element contain orphan inline nodes.
+        // If so we transform the base element HTML to put those orphans
+        // inside `<p>` containers.
+        const orphanInlineChildNodes = [...element.childNodes].filter(
+            (n) => !isBlock(n) && (n.nodeType === Node.ELEMENT_NODE || n.textContent.trim() !== "")
+        );
+        if (orphanInlineChildNodes.length) {
+            let currentP;
+            const childNodes = [...element.childNodes];
+            const tempEl = document.createElement('temp-container');
+            const appendAndRenewCurrentP = () => {
+                if (currentP && currentP.childNodes.length > 0 && currentP.innerHTML.trim() !== '') {
```

Comment:
Why the `!== ''` condition ? What special case is it supposed to catch ? If it really makes sense then this line should at least have a comment ^^'

Reply:
I'll add a comment to explain.